lambdas/api/template.yaml

```diff
@@ -1030,6 +1010,66 @@ Resources:
       # Explicit retention time
       RetentionInDays: !Ref LogRetentionDays
 
+  # Get DMP metadata files
+  PutUploadsDmpsFunction:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      # CodeUri: 'put_dmps_uploads/dist/'
```

Comment:
L1017: Do we still need this commented line?

Reply:
Oh great catch. That commented out line is the correct one. I was trying to get it to compile from the Typescript source but it fails. I forgot to switch it back before pushing